14-microbiome-diversity.Rmd

```diff
@@ -142,6 +184,8 @@ plotReducedDim(se, "PCA", colour_by = "SampleType")
 As mentioned before, applicability of the different methods depends on your
 sample set.
 
+FIXME: let us switch to UMAP for the examples?
```

Comment:
Maybe we can do a comparison? The switch to UMAP is fine with me as well

Reply:
At least there are other papers that compare t-SNE and UMAP; the latter is newer and much faster and should be the default in general imo.